applications/facings-nft-collection-manager.md

```diff
@@ -0,0 +1,233 @@
+
+### Overview
+
+- **Name:** NFT Collection Manager
+- **Brief Description:** An interface to the AtomicAssets standard which makes it easy to manage your own Atomic NFT Collections, with draft mode and collaboration features to ensure high quality releases and gamification integration.
+- **Relationship to EOSIO:** FACINGS has been launching AtomicAssets NFTs  since its inception in early 2021. Its parent company and initial investor, Detroit Ledger Technologies (formerly EOS DETROIT), has been in the EOSIO ecosystem since its inception as a pioneer block producer on EOS and other EOSIO blockchains.
+- **Reason for Interest:** Our team and experience combined with the need for an open-source creator that is friendlier to use makes for a perfect fit. The lack of development on EOS for NFTs and gamification resources makes it a necessity going forward.
+
+
+### Project Details
+
+**Mock-ups/designs of any UI components**
+
+These designs are from past prototypes, to give an example, but does not represent the final product: 
+https://drive.google.com/drive/folders/1iN2bQJhB_w_A_fKXC8-iYr3Re2-rbqSt
+
+**Data models of the core functionality**
+
+The data model will be largely based on the AtomicAssets [schema](https://github.com/pinknetworkx/atomicassets-contract/wiki/Tables).
+
+**API specifications of the core functionality**
+
+We seek to implement a large portion of the [AtomicAssets](https://github.com/pinknetworkx/atomicassets-contract/wiki/Actions) allowed actions in the NFT Collection Manager UI bringing together the power of existing actions into the EOS framework for implementation with complex collections and gamification.
+
+
+**An overview of the technology stack to be used**
+
+The NFT Collection Manager will use React/JS. Our goal is to minimize dependencies. Therefore we intend for all of the data to be stored on-chain. In the future, IPFS will be used to store all NFT images and a testnet is used to stage draft collections.
+
+The NFT Collection Manager will be a client-side app built in React and talking directly with public blockchain API nodes.
+
+
+**Documentation of core components, protocols, architecture, etc. to be deployed**
+
+- Base Implementation (MVP/Core functionality)
+  - Home page: featured collections and an “about” section
+- Login functionality. When logged in, show:
+  - Logged in user’s account name and profile photo
+  - Option to log out
+  - RAM/Resource usage
+- Account page: list user’s collections
+- Basic collection viewer (Collection, Schemas, Templates, Assets)
+- Basic collection manager
+  - Create/Edit Collections
+  - Create/Extend Schemas
+  - Create Templates
+  - Create Assets
+  - Lock template
+  - Burn asset
+- Basic Collection explorer
+  - Discovery features (trending or featured collections)
```

Comment:
The discovery features are presumably part of a future phase, though "featured collections" are mentioned in this phase's scope of work.

The target audience is:
> Our target audience consists of game developers, members of the current EOS and derivative chains. We also hope to encourage members of other chains to create on EOS in the future. By leveling the playing field for EOS with improved tooling, there is a compelling argument for game producers and collection enthusiasts to develop and launch on EOS.

Discovery features seem to target end-consumers of NFTs. 

Is the long term vision that this competes with other NFT Marketplaces which have some basic tools to create NFTs/Collections _OR_ is this a developer tool? If it is the latter, it seems like some of these features are distractions. Marketplaces/social-networks will be a more natural place for discovery, and the future "automatic collection microsite generation (configurable sales sites)" seem like the natural place to focus on the ergonomics of collection viewing/indexing.

Furthermore, the opensource portion that serves the vision of this grant proposal should be able to stand alone from the greater FACINGS business opportunity. 

I would like to better understand the separation between the tool(s) that are great fits for grant funding and the for-profit side of the business.

Reply:
Our vision for FACINGS is as an NFT launchpad for collection owners and publishers. That means we aim to assist with every aspect that we can to improve the success of NFT launches, and consequently the publishers’ overall collections.

The initial discovery features outlined for the home page are meant to be more informational and create a lively and engaging experience for visiting traffic, including the publishers. These aren’t intended to be deep features at this time.

As far as our roadmap pertains to marketplaces, we don’t have any intentions of creating a new secondary marketplace at this time. However we do foresee creating integrations into  pre-existing secondary marketplace smart contracts such as atomicmarket. Some examples of this would be providing a listings page on a collection’s microsite to show that collection’s listings in the future, or participating in atomicmarket’s maker model to earn transaction fees for listings purchased on our own UI. Both of these examples are outside of the scope of this proposal.

We agree that the open source portion developed by the grant should not rely on any FACINGS dependencies. Our current thinking is to develop the codebase towards a plugin architecture in order to promote development of new features, integration of more NFT standards, and to create a layer above the core codebase for businesses to thrive. This model would create room for businesses to innovate while allowing us to build up a shared core platform fully open source and funded by the ENF.